<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelescopeClientINDIUI</class>
 <widget class="QWidget" name="TelescopeClientINDIUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="serverSettings">
     <property name="title">
      <string>Connection settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutConnectionSettings">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelHost">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEditHostName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Host name or IPv4 address of the machine that hosts the telescope server</string>
        </property>
        <property name="text">
         <string notr="true">localhost</string>
        </property>
        <property name="maxLength">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelColon">
        <property name="text">
         <string notr="true">:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBoxTCPPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelTCPPort">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>TCP port:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="getDevicesPushButton">
     <property name="text">
      <string>Check devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="devicesComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
